src/Components/test/testassets/BasicTestApp/CulturePicker.razor

```diff
@@ -0,0 +1,18 @@
+@inject IUriHelper UriHelper
+
+<h3>Select your language</h3>
+<select @onchange="@OnSelected" id="culture-selector">
+    <option>Select...</option>
+    <option value="en-US">English</option>
+    <option value="fr-FR">Français</option>
```

Comment:
I debated using `en-GB` or `en-CA` or even `en-NZ` - but French seems like a better choice since their number format is significantly different.

The next step here is to add tests that verify number/date parsing.

Reply:
C'est un choix judicieux